workflows/execute_workflows.sh

```diff
@@ -119,7 +123,7 @@ do
         cp "$WORKFLOW" "$WS_DIR"
     done
 
-    DIR_NAME=$(echo $WS_DIR | rev | cut -d'/' -f 2 | rev)
+    DIR_NAME=$(dirname $WS_DIR)
```

Comment:
```suggestion
    DIR_NAME=$(echo $WS_DIR | rev | cut -d'/' -f 2 | rev)
```

Reply:
See above

Reply:
Try `s dirname basename `.

Reply:
`$(basename $WS_DIR)` works fine, so I updated the suggestion.